.github/workflows/links.yml

```diff
@@ -23,7 +24,7 @@ jobs:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
 
       - name: Create Issue From File
-        if: env.exit_code != 0
+        if: ${{ steps.lychee.outputs.exit_code }} != 0
```

Comment:
@YDX-2147483647, this seems to have lead to a false-positive issue creation here: https://github.com/lycheeverse/lychee-action/issues/264

Note that there were no issues with the check run, but it still created the issue.

Reply:
Oh, sorry… I guess `${{ steps.lychee.outputs.exit_code }} != 0` evaluates to `"0 != 0"` (a string), and becomes `true` when GitHub coerces its type.

So it should be `if: steps.lychee.outputs.exit_code != 0` or `if: ${{ steps.lychee.outputs.exit_code != 0 }}`. Not sure, however. I'll test it in my own repo later (in one or two days).

Reply:
I have just tested it, and it works for both success and failure. Please see #265.